addons/web/static/src/js/views/list/list_renderer.js

```diff
@@ -366,6 +366,9 @@ var ListRenderer = BasicRenderer.extend({
             },
         });
         _.each(this.arch.children, function (c) {
+            if (c.tag === 'header') {
+                return;
+            }
```

Comment:
I'd go for `if (c.tag !== 'control' && c.tag !== 'groupby' && c.tag !== 'header') {` for the sake of consistency.
Or with a bit of refactoring, something like:
```js
this.arch.children
    .filter(c => !['control', 'groupby', 'header'].includes(c.tag))
    .forEach(c => { /* ... */ });
```